code/_onclick/item_attack.dm

```diff
@@ -41,6 +41,15 @@
 	return FALSE
 
 /obj/attackby(obj/item/I, mob/living/user, params)
+	var/turf/T = (get_turf(src))
+	if(user.a_intent == INTENT_HARM && user.m_intent == MOVE_INTENT_WALK)//if you swing carefully without blocking, you click easier
+		for(var/mob/living/L in T)
+			if(L.stat)
```

Comment:
You sure about this?

Reply:
Yeah. There's no need for aiming if they cant dodge, and you want to hit the guy with a gun, not the softcrit dude he's standing on